FIC-Scheduler/server/src/main/resources/application.properties

```diff
@@ -1 +1,10 @@
+server.tomcat.accesslog.enabled=true
 
+
+spring.jpa.hibernate.ddl-auto=update
+spring.jpa.show-sql=true
+
+spring.datasource.url=jdbc:postgresql://dpg-chtdp4m7avj345epjl6g-a.oregon-postgres.render.com/database_vy4u
+spring.datasource.username=database_vy4u_user
+spring.datasource.password=aM8AlM28g9hjfEnEyqq0lJpU1f1nUhkB
```

Comment:
Ideally DB credentials should be configured via config vars! Something to fix imminently